### DISCUSSION TITLE
Shared login with Django

### DISCUSSION THREAD

**Discussion by praxidian** [LiveHelperChat/livehelperchat#2055] [Q&A]
Hi there, what would be the best solution to share LHC authentication with a Django app? Can this be done via API authentication? I would like to build a Django platform that passes logged-in users to LHC without requiring any manual login action.

**Comment by remdex** [comment]
You can try to looks at those
* https://github.com/LiveHelperChat/sso-boilerplate
* https://doc.livehelperchat.com/docs/users/auto-login
* https://github.com/LiveHelperChat/lhcgoogleauth

Those scenarios depends how you will authorise user. Should it already exists in LHC or not. Should it be created? if so in what group it should end up.

**Reply by praxidian** [reply]
Many thanks, this is a fantastic starting point.
